rubygem-foreman_templates/rubygem-foreman_templates.spec

```diff
@@ -89,6 +94,13 @@ GEMFILE
 %doc %{gem_instdir}/LICENSE
 %doc %{gem_instdir}/README.md
 
+%posttrans
+%{foreman_db_migrate}
```

Comment:
Other plugins with this have DB migrations and seeds to run, this plugin doesn't so they should be removed to improve installation speed.